flake.nix

```diff
@@ -5,8 +5,7 @@
     flake-parts.follows = "common/flake-parts";
     systems.url = "github:nix-systems/default";
 
-    # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/e355d4aae4a46283bf0a01d4c6f38bd01153b3c2";
```

Comment:
shared-kernel should be updated without changing `flake.nix`; see https://github.com/nammayatri/nammayatri/tree/main/Backend#updating-flake-inputs